packages/components/src/components/Text/Text.tsx

```diff
@@ -79,6 +79,9 @@ export const Text: FC<TextProps> = ({
   variant = 'body',
 }) => {
   const theme = useTheme()
+
+  if (!children) return null
```

Comment:
Was this needed for TS to not get upset about the bold check or just a random piece of "defensive" code?